release-notes/7.0/releases.json

```diff
@@ -3,7 +3,7 @@
   "latest-release": "7.0.5",
   "latest-release-date": "2023-04-11",
```

Comment:
this field was updated on releases-index but not here. which one is correct? and is this field applicable to runtime or SDK release date?

Reply:
https://github.com/dotnet/core/pull/8495